controller/src/main/java/io/pravega/controller/store/stream/Stream.java

```diff
@@ -308,9 +308,11 @@
 
     /**
      * Returns the currently active stream epoch.
+     *
+     * @param ignoreCached if ignore cache is set to true then fetch the value from the store. 
      * @return currently active stream epoch.
      */
-    CompletableFuture<Pair<Integer, List<Integer>>> getActiveEpoch();
+    CompletableFuture<Pair<Integer, List<Integer>>> getActiveEpoch(boolean ignoreCached);
```

Comment:
This parameter is a bit obscure. It is clear what it does, but not clear when to set or unset it.